diffusion_with_autoscaler/autoscaler.py

```diff
@@ -598,20 +601,22 @@ def __init__(
         self._last_autoscale = time.time()
         self.fake_trigger = 0
 
+        for _ in range(min_replicas):
```

Comment:
For another PR, we need to clean this out with structures. This isn't begineer friendly.